bc_obps/registration/fixtures/real/naicsCode.json

```diff
@@ -0,0 +1,230 @@
+[
+  {
+    "model": "registration.naicsCode",
+    "pk": 1,
+    "fields": {
+      "id": 1,
+      "naics_code": "211110",
+      "naics_description": "Oil and gas extraction (except oil sands)"
+    }
+  },
+  {
+    "model": "registration.naicsCode",
+    "pk": 2,
+    "fields": {
+      "id": 2,
+      "naics_code": "212114",
+      "naics_description": "Bituminous coal mining"
+    }
+  },
+  {
+    "model": "registration.naicsCode",
+    "fields": {
+      "naics_code": "212220",
```

Comment:
FYI, the naics code dropdown in the form shows the ids, not the code or the code title. We've got a ticket to address that already: https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/318